contracts/adapters/linkswap/LinkswapAdapter.sol

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2020 Zerion Inc. <https://zerion.io>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import { ERC20 } from "../../ERC20.sol";
+import { ProtocolAdapter } from "../ProtocolAdapter.sol";
+
+
+/**
+ * @title Adapter for LINKSWAP protocol.
+ * @dev Implementation of ProtocolAdapter interface.
+ * @author Giovanni Di Siena <giodisiena@gmail.com>
+ */
+contract LinkswapAdapter is ProtocolAdapter {
+
+    string public constant override adapterType = "Asset";
+
+    string public constant override tokenType = "LINKSWAP LP Token (LSLP)";
```

Comment:
if LSLP has the same interface as UniswapV2 LP tokens, there is no need to redeploy TokenAdapter, just use `"Uniswap V2 pool token"` as `tokenType` and it will be working)

Reply:
I may be misunderstanding, but I believe these are the interfaces you are referring to: [IUniswapV2Pair.sol](https://github.com/Uniswap/uniswap-v2-core/blob/master/contracts/interfaces/IUniswapV2Pair.sol), [ILinkswapPair.sol](https://github.com/yflink/linkswap/blob/master/contracts/interfaces/ILinkswapPair.sol)? In which case they differ, so I think this distinction is necessary?

Reply:
@cr-igor okay, so does that mean that I will need to remove references to `LinkswapTokenAdapter` [here](https://github.com/zeriontech/defi-sdk/pull/134/files#diff-7d6910f4d9b0ea604c8a7575be91ff6fce9083b589736da559e0b11ecaa712c9) in `1_deploy_registry_and_add_adapters.js`? Would this include L1685?

Reply:
yes, you should remove LinkswapTokenAdapter from this file as well as L1685
and fix tokenType in LinkswapAdapter.sol